Microsoft.Azure.Devices.Edge.sln

```diff
@@ -213,9 +213,11 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TwinTester", "test\modules\
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DeploymentTester", "test\modules\DeploymentTester\DeploymentTester.csproj", "{1C1C8203-CD73-4CC1-9112-7315F9DE1AB6}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.Devices.Edge.Agent.Integration.Test", "edge-agent\test\Microsoft.Azure.Devices.Edge.Agent.Integration.Test\Microsoft.Azure.Devices.Edge.Agent.Integration.Test.csproj", "{B3C68519-B309-42FF-B4DF-9E77B2D4BD50}"
```

Comment:
make sure this doesn't cause any change automatically when open by vs code or vs 2019.

Reply:
I user VS2019 to generate this.

Reply:
try open it in VS Code to verify.

Reply:
Done.

Reply:
This kept happening on my IDE too (Visual Studio). I would change it back manually before checking in. I guess it really wants to change it, though...